.cookiecutter/includes/tox/passenv

```diff
@@ -0,0 +1 @@
+HUBSPOT_API_KEY
```

Comment:
`HUBSPOT_API_KEY` is [currently in `tox.ini`'s `passenv`](https://github.com/hypothesis/report/blob/f309dde299884d2e6370f96fda896c6ea7f5d069/tox.ini#L31). The cookiecutter requires you to put it in this include file.